### PR TITLE
Visualization Tutorial/Example

### PR DESCRIPTION
**Pull request by danlamanna**
@jeffbaumes Here's a rough draft of the tutorial we spoke about.

I ran into a few issues using port/spec functionality, namely:
- Using a direct callable instead of a script tag bypasses romanesco.run which means auto-conversion isn't currently supported in that manner
- ```romanesco.load``` doesn't appear to be integrated into Task yet, so ```script_uri``` within a Task specification won't be fetched
- I ran into issues using Ports because romanesco.run expects the inputs to exist on the task dictionary as well.

I didn't spend too much time looking into them because I'm not sure where we're headed with this functionality - but I'm sure they could be resolved without a large amount of effort.

### CODE REVIEW

**Comment by jeffbaumes** [comment]
This is looking very good. You should be able to build a workflow with JSON based on the two analysis specs, instead of running them separately. Look at the JSON spec documentation for a WORKFLOW_TASK to see how to do it.

**Comment by danlamanna** [comment]
@jeffbaumes I updated this to use the WORKFLOW builtins, local URL to view it still works.

One thing I noticed was auto-conversion only exists on input, not on output. We could skip the ```convert``` step at the bottom if we specified the workflow output as ```networkx.json``` and it auto-converted.

**Comment by jeffbaumes** [comment]
LGTM after you remove the call to `convert` as we discussed.

File: docs/examples.rst [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+       475     484
+
+Each integer represents an anonymized Facebook user. Users belonging to the same line in the adjacency list indicates a symmetric relationship in our undirected graph.
+
+Build a Workflow
+~~~~~~~~~~~~~~~~
+
+Create a file named ``workflow.py``, this is the file we'll be using to create our Romanesco workflow.
+
+Finding the Most Popular Person
```
Comment (ronichoudhury) [inline docs/examples.rst]: s/Finding/Find/

File: docs/examples.rst [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+.. literalinclude:: static/facebook-example.py
+    :caption: workflow.py/most_popular_task
+    :lines: 1-22
+    :linenos:
+
+.. note :: As we saw with our last script assuming ``G`` would in be scope, this task explicitly states that both ``most_popular_person`` and ``G`` will be in scope (as its outputs) when it's done.
+
+
+Finding Their Neighborhood
```
Comment (ronichoudhury) [inline docs/examples.rst]: s/Finding/Find/

File: docs/examples.rst [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+.. literalinclude:: static/facebook-example.py
+    :caption: workflow.py/find_neighborhood_task
+    :lines: 24-44
+    :lineno-start: 27
+    :linenos:
+
+
+Putting It Together
+###################
```
Comment (ronichoudhury) [inline docs/examples.rst]: s/Putting/Put/

**Comment by ronichoudhury** [comment]
This looks great!

**Pull request merged**